collective-big-count/test_exscan.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2022 IBM Corporation.  All rights reserved.
+ *
+ * $COPYRIGHT$
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <mpi.h>
+#include "common.h"
+
+int my_c_test_core(MPI_Datatype dtype, size_t total_num_elements, bool blocking);
+
+int main(int argc, char** argv) {
+    /*
+     * Initialize the MPI environment
+     */
+    int ret = 0;
+
+    MPI_Init(NULL, NULL);
+    init_environment(argc, argv);
+
+    // Run the tests
+#ifndef TEST_UNIFORM_COUNT
+    // Each rank contribues: V_SIZE_INT elements
+    // Largest buffer is   : V_SIZE_INT elements
+    ret += my_c_test_core(MPI_INT, V_SIZE_INT, true);
+    ret += my_c_test_core(MPI_C_DOUBLE_COMPLEX, V_SIZE_DOUBLE_COMPLEX, true);
+    if (allow_nonblocked) {
+        ret += my_c_test_core(MPI_INT, V_SIZE_INT, false);
+        ret += my_c_test_core(MPI_C_DOUBLE_COMPLEX, V_SIZE_DOUBLE_COMPLEX, false);
+    }
+#else
+    size_t proposed_count;
+
+    // Each rank contribues: TEST_UNIFORM_COUNT elements
+    // Largest buffer is   : TEST_UNIFORM_COUNT elements
+    proposed_count = calc_uniform_count(sizeof(int), TEST_UNIFORM_COUNT,
+                                        2, 2, 1.0); // 1 send, 1 recv buffer each
```

Comment:
when you rebase you will need to remove the last parameter from the `calc_uniform_count` commands:
```suggestion
                                        2, 2); // 1 send, 1 recv buffer each
```

Reply:
The MPI_Alltoallv test is complete, so implementation of this pull request is complete